modules/ROOT/partials/security/sanitizing-html-input-and-protecting-against-xss-attacks-dom-parser-and-dom-purify.adoc

```diff
@@ -0,0 +1,68 @@
+[[sanitizing-html-input-to-protect-against-xss-attacks]]
+=== Sanitizing HTML input to protect against XSS attacks
+
+Previously, before HTML content was passed to {productname} 5.x, it was parsed using an https://tiny.cloud/docs/api/tinymce.html/tinymce.html.saxparser/[in-house developed API], `SaxParser`.
```

Comment:
If we ever get around to finishing that migration of the 5x docs to antora, the link will break.

A link to the branch is also not guaranteed - you'd need to link to a _commit_ tree for that - but I think it's more likely to be valid long-term.

The docs were rendered from the code so all the same information is available.

```suggestion
Previously, before HTML content was passed to {productname} 5.x, it was parsed using an https://github.com/tinymce/tinymce/blob/release/5.10/modules/tinymce/src/core/main/ts/api/html/SaxParser.ts[in-house developed API], `SaxParser`.
```